src/pages/home/report/ReportActionItem.js

```diff
@@ -160,4 +182,7 @@ export default withOnyx({
     betas: {
         key: ONYXKEYS.BETAS,
     },
+    report: {
+        key: ({reportID}) => `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
+    },
```

Comment:
If we already have `reportID`, isn't this unnecessary -- now that we simply pass reportID

Reply:
This is `chatReportID` but we need `iouReportID` and we need to fetch chatReport first to use `report.iouReportID` in child component's withOnyx. I'm now passing report to use `report.hasOutstandingIOU` anyway.

Reply:
Can we pass the `iouReportID` or anything else we need directly?